Source/OxyPlot/Axes/Axis.cs

```diff
@@ -543,6 +543,11 @@ public double Scale
         public bool UseSuperExponentialFormat { get; set; }
 
         /// <summary>
+        /// Gets or sets the natural dimensions of the axis, as of most recent rendering.
+        /// </summary>
+        public OxySize NaturalSize { get; protected set; }
```

Comment:
Right, the "ActualX" properties should indicate the actual size that was chosen for rendering.
Sorry, I don't get the purpose of this new property yet...